test/Quake/arg_subst-5.txt

```diff
@@ -0,0 +1,11 @@
+// ========================================================================== //
```

Comment:
Should we name these `.txt` files with an extension like `.subst` to help identify them?